mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -461,6 +461,13 @@ public boolean onMenuItemClick(MenuItem menuItem) {
                 }
             });
             //inflateClientsAccounts();
+        } else {
+            Toaster.show(getView(), R.string.failed_to_load_client_details);
```

Comment:
The view should not contain the business logic. This check should happen in the presenter and then presenter should then call another method on the view to hide the progress bar.